libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -63,6 +64,8 @@ @interface SFHybridViewController()
     SFHybridViewConfig *_hybridViewConfig;
 }
 
+@property (nonatomic, readwrite, assign) BOOL useWKWebView;
```

Comment:
I know it might be overkill...but should we consider subclasses of `SFHybridViewController` that address the different web views being used?